OptionalCommand/Jackalope/InitDoctrineDbalCommand.php

```diff
@@ -47,7 +47,19 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        DoctrineCommandHelper::setApplicationConnection($this->getApplication(), $input->getOption('session'));
+        $application = $this->getApplication();
+        $sessionName = $input->getOption('session');
+        if (empty($sessionName)) {
+            $container = $application->getKernel()->getContainer();
+            $sessionName = $container->getParameter('doctrine_phpcr.admin_session')
+                ?: $container->getParameter('doctrine_phpcr.default_session');
+
+            if (empty($sessionName)) {
+                // throw new RuntimeException("No session name could be determined. This command requires either that the 'session' option is passed to the command or an 'admin_session' is configured in the DoctrinePHPCRBundle configuration, with a workspace name that must exist ('default' is recommended)");
```

Comment:
I was wondering to throw an exception here and not default to ``default_session`` above .. but I think this will just cause confusion and break all our builds.